RevenueCatUI/Views/FooterView.swift

```diff
@@ -71,9 +71,13 @@ private struct RestorePurchasesButton: View {
             _ = try await self.purchaseHandler.restorePurchases()
             self.displayRestoredAlert = true
         } label: {
-            ViewThatFits {
+            if #available(iOS 16.0, macOS 13.0, tvOS 16.0, *) {
+                ViewThatFits {
+                    Text("Restore purchases", bundle: .module)
+                    Text("Restore", bundle: .module)
+                }
+            } else {
                 Text("Restore purchases", bundle: .module)
-                Text("Restore", bundle: .module)
```

Comment:
Is the "Restore" not needed for iOS 15? 🤔

Reply:
This is an optimization to make it look better if it doesn't fit. We can't use this in iOS 15 so we gotta rely on `Text` clipping the end of the word if it doesn't fit.